### PR TITLE
feat(ui): Relocate resource filters

### PR DESCRIPTION
**Pull request by rbreeze**
Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] Optional. My organization is added to USERS.md.
* [ ] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

File: ui/src/app/applications/components/application-details/application-resource-filter.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+                )}
+                <ActionButton
+                    label={!shown ? 'SHOW' : 'HIDE'}
+                    action={() => setShown(!shown)}
+                    style={{marginLeft: pref.resourceFilter.length > 0 ? '5px' : 'auto', fontSize: '12px', lineHeight: '5px', display: !shown && 'block'}}
+                />
+            </div>
+            {shown && (
+                <div className='applications-list__filters'>
+                    {ResourceFilter({label: 'OWNERSHIP', prefix: 'ownership', options: ['Owners', 'Owned']})}
```
Comment (alexmt) [inline ui/src/app/applications/components/application-details/application-resource-filter.tsx:104]: I think we should move the most frequently used filters to the top. Can you change order as following:

* KINDS
* SYNC STATUS
* OWNERSHIP
* CREATED WITHIN (rename it to AGE)

File: ui/src/app/applications/components/application-details/application-resource-filter.tsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+                    {ResourceFilter({label: 'SYNC STATUS', prefix: 'sync', options: ['Synced', 'OutOfSync']})}
+                    {ResourceFilter({label: 'HEALTH STATUS', prefix: 'health', options: ['Healthy', 'Progressing', 'Degraded', 'Suspended', 'Missing', 'Unknown']})}
+                    {ResourceFilter({label: 'KINDS', prefix: 'kind', options: kinds, field: true})}
+                    {namespaces.length > 1 && ResourceFilter({label: 'NAMESPACES', prefix: 'namespace', options: (namespaces || []).filter(l => l && l !== ''), field: true})}
+                    {ResourceFilter({label: 'CREATED WITHIN', prefix: 'createdWithin', options: ['1m', '3m', '5m', '15m', '60m'], radio: true})}
```
Comment (alexmt) [inline ui/src/app/applications/components/application-details/application-resource-filter.tsx:109]: Let rename `CREATED WITHIN` to age please

**Review by alexmt — CHANGES_REQUESTED** [review]
Added couple comments.

File: ui/src/app/applications/components/application-details/application-resource-filter.tsx (hunk 2 above)
Comment (rbreeze) [reply ui/src/app/applications/components/application-details/application-resource-filter.tsx:109]: That's a great idea, `CREATED WITHIN` was long and pushed the clear button to a separate row

**Review by alexmt — APPROVED** [review]
LGTM

**Pull request merged**